lib/src/prices/get_challenge_list.dart

```diff
@@ -0,0 +1,69 @@
+import '../utils/json_helper.dart';
+
+/// Get Challenge List
+class GetChallengeList {
```

Comment:
Should probably be renamed `GetChallengesResult` (file and class), in order to match existing classes like `GetPriceProductsResult`